.travis.yml

```diff
@@ -1,4 +1,7 @@
 language: c
+compiler: gcc
+sudo: require
+dist: xenial # Oldest still-supported LTS right now against which the AppImage is built. Update this to the next LTS when support drops.
```

Comment:
Don't these three settings belong to the AppImage build only?

Reply:
Can `sudo: require` be actually moved in the matrix?
Feel free to do these changes directly on my branch.